# alignment file autogenerated by Paul's awesome script 
# 03:27PM on March 04, 2018
# ========================================================
a_s0_roll:  -2.913
a_s2_roll:  -1.924
a_s1_pitch:  -0.032
a_s3_roll:  0.000
a_s3_yaw:  3.141
a_s3_pitch:  0.635
a_s2_yaw:  -1.574
a_s1_z:  1.756
a_s1_y:  0.353
a_s1_x:  -0.035
a_s0_z:  1.773
a_s0_x:  0.053
a_s0_y:  -0.210
a_s3_y:  -0.001
a_s3_x:  -0.241
a_s3_z:  1.823
a_s1_yaw:  -0.739
a_c0_x:  0.000
a_c0_y:  -0.019
a_s2_x:  -0.049
a_s2_y:  0.012
a_s2_z:  1.153
a_s1_roll:  -2.881
a_c0_pitch:  -0.009
a_s2_pitch:  -0.006
a_c0_yaw:  -1.645
a_s0_pitch:  0.025
a_c0_z:  1.710
a_c0_roll:  -1.890
a_s0_yaw:  -2.466
